covigator/database/model.py

```diff
@@ -1150,3 +1152,43 @@ class Lineages(Base):
     variant_under_investigation = Column(Boolean, default=False)
     parent_lineage_id = Column(String)
     tags = Column(String)
+
+    variants = relationship(
+        "LineageDefiningVariants",
+        secondary=LINEAGE_SITES_JUNCTION_TABLE_NAME,
+        back_populates='constellations',
+    )
+
+
+class LineageDefiningVariants(Base):
+    """
+    Store lineage defining mutations
+    """
+    __tablename__ = CONSTELLATION_SITES_TABLE_NAME
+
+    variant_id = Column(String, primary_key=True)
+    variant_type = Column(String)
+    protein = Column(String)
+    position = Column(Integer, index=True)
```

Comment:
If I understand it right the position, reference and alternate may be  in the genomic space or in the proteomic space. Please, add some documentation lines here for our future selves or anyone else.